osm_validator/views.py

```diff
@@ -49,4 +61,31 @@
             await conn.execute(models.User.__table__.update().
                                where(models.User.__table__.c.osm_uid == user['osm_uid']).
                                values(**user))
+
+    await request.app.redis.oauth.set(
+        'user',
+        user['osm_uid'],
+        expire=request.app.config.OAUTH_CACHE_EXPIRE)
+
+    session['user'] = user['osm_user']
+    session['message'] = user['osm_uid']
```

Comment:
lets put to session `session['user_id'] = user['osm_uid']` it's unique and unchangeable